js/form.js

```diff
@@ -0,0 +1,31 @@
+const adForm = document.querySelector('.ad-form');
+const adFormInteractive = [...adForm.children];
+const mapFilters = document.querySelector('.map__filters');
+const mapFilterInteractive = [...mapFilters.children];
+
+
+const putInactiveState = () => {
+  adForm.classList.add('.ad-form--disabled');
+  adFormInteractive.forEach((element) => {
+    element.disabled = true;
```

Comment:
лучше использовать setAttribute
```suggestion
    element.setAttribute('disabled', true);
```